src/styles/blocks/card.css

```diff
@@ -0,0 +1,87 @@
+.card {
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 32px 16px 19px;
+}
+
+.card__img {
+  display: block;
+  width: 160px;
+  height: 134px;
+  margin: 0 auto ;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin: 40px 0 0;
```

Comment:
check the previous comment https://github.com/mate-academy/layout_product-cards/pull/2738#discussion_r1138309451